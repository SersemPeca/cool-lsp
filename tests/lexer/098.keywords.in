case
elif
else
if
class
fi
in
inherits
isvoid
let
loop
pool
then
while
case
esac
new
of
not
