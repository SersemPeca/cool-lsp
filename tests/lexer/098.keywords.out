#1 CASE
#2 OBJECTID elif
#3 ELSE
#4 IF
#5 CLASS 
#6 FI
#7 IN
#8 INHERITS
#9 ISVOID
#10 LET
#11 LOOP
#12 POOL
#13 THEN
#14 WHILE
#15 CASE
#16 ESAC
#17 NEW
#18 OF
#19 NOT
